### ISSUE TITLE
Build failure with GHC 9.4

### ISSUE THREAD

**Issue by andreasabel** [Lysxia/generic-data#57]
Build is broken with GHC 9.4.1:
```
[14 of 19] Compiling Generic.Data.Internal.Generically ( src/Generic/Data/Internal/Generically.hs, /Users/abel/bin/src/generic-data/dist-newstyle/build/x86_64-osx/ghc-9.4.1/generic-data-0.9.2.1/build/Generic/Data/Internal/Generically.o, /Users/abel/bin/src/generic-data/dist-newstyle/build/x86_64-osx/ghc-9.4.1/generic-data-0.9.2.1/build/Generic/Data/Internal/Generically.dyn_o )

src/Generic/Data/Internal/Generically.hs:73:32: error:
    Ambiguous occurrence ‘Generically’
    It could refer to
       either ‘GHC.Generics.Generically’,
              imported from ‘GHC.Generics’ at src/Generic/Data/Internal/Generically.hs:24:1-19
           or ‘Generic.Data.Internal.Generically.Generically’,
              defined at src/Generic/Data/Internal/Generically.hs:71:1
   |
73 | instance Generic a => Generic (Generically a) where
   |                                ^^^^^^^^^^^
```
and many more of these...

**Comment by andreasabel** [comment]
@Lysxia, see https://downloads.haskell.org/~ghc/9.4.1/docs/users_guide/9.4.1-notes.html#base-library
> GHC.Generics now provides a set of newtypes, `Generically` and `Generically1`, for deriving generic instances via [:lang-ext:`DerivingVia`](https://downloads.haskell.org/~ghc/9.4.1/docs/users_guide/9.4.1-notes.html#id7). `Generically` instances include `Semigroup` and `Monoid`.

**Comment by Lysxia** [comment]
Thanks for the report. #58 should fix this. Will release over the week end.

**Comment by Lysxia** [comment]
Released: [1.0.0.0](https://hackage.haskell.org/package/generic-data)
